commands/pug.py

```diff
@@ -144,11 +153,15 @@ def get_char(name, server):
 
 
 async def pug(client, message):
+    target_region = default_region
```

Comment:
Put this and line 156 in the Try: block. They shouldn't fail, but the block is to ensure some kind of bot output is always received.